### PR TITLE
Added support for 2.12, custom namespaces support and Enterprise-dr product

### PR DESCRIPTION
**Pull request by pragrawal-px**
Signed-off-by: pragrawal10 <pragrawal@purestorage.com>

**What this PR does / why we need it**:

**Which issue(s) this PR fixes** (optional)
Closes #

**Special notes for your reviewer**:

### CODE REVIEW

**Comment by Glitchfix** [comment]
LGTM

**Review by Glitchfix — APPROVED** [review]

File: locals.tf [hunk 1]
```diff
@@ -30,27 +36,28 @@ locals {
...
+    pxOperatorImageVersion = "1.10.0"
     envVars                = ""
     maxStorageNodesPerZone = 3
     useOpenshiftInstall    = false
     etcdEndPoint           = ""
     dataInterface          = ""
     managementInterface    = ""
     useStork               = true
-    storkVersion           = "2.11.0"
+    storkVersion           = "2.12.0"
```
Comment (sharma-tapas) [inline locals.tf]: 2.12.2

File: locals.tf [hunk 2]
```diff
@@ -30,27 +36,28 @@ locals {
...
 
   default_helm_values = [templatefile("${path.module}/values.yaml", {
-    imageVersion           = "2.11.0"
+    imageVersion           = "2.12.0"
     clusterName            = local.name
-    drives                 = "type=gp2,size=200"
+    drives                 = "type=gp2+size=200"
     useInternalKVDB        = true
     kvdbDevice             = "type=gp2,size=150"
+    pxOperatorImageVersion = "1.10.0"
```
Comment (sharma-tapas) [inline locals.tf]: 1.10.3

File: locals.tf [hunk 3]
```diff
@@ -30,27 +36,28 @@ locals {
...
+    create_kubernetes_namespace       = local.create_namespace
     kubernetes_namespace              = local.namespace
     create_kubernetes_service_account = true
     kubernetes_service_account        = "${local.service_account_name}"
     irsa_iam_policies                 = local.irsa_iam_policies_list
   }
 
   default_helm_values = [templatefile("${path.module}/values.yaml", {
-    imageVersion           = "2.11.0"
+    imageVersion           = "2.12.0"
```
Comment (sharma-tapas) [inline locals.tf]: 2.12.2

File: locals.tf [hunk 4]
```diff
@@ -16,8 +22,8 @@ locals {
     name        = local.name
     description = "A Helm chart for portworx"
     chart       = "portworx"
-    repository  = "https://raw.githubusercontent.com/portworx/eks-blueprint-helm/main/repo/stable"
-    version     = "2.11.0"
+    repository  = "https://raw.githubusercontent.com/portworx/eks-blueprint-helm/namespace/repo/beta"
+    version     = "2.12.0"
```
Comment (sharma-tapas) [inline locals.tf]: 2.12.2

**Review by sharma-tapas — APPROVED** [review]

**Review by Glitchfix — APPROVED** [review]

**Pull request merged**